packages/pigeon/platform_tests/test_plugin/example/ios/RunnerTests/AsyncHandlersTest.swift

```diff
@@ -43,9 +42,8 @@ class AsyncHandlersTest: XCTestCase {
 
     let expectation = XCTestExpectation(description: "voidvoid callback")
     binaryMessenger.handlers[channelName]?(nil) { data in
-      let outputMap = binaryMessenger.codec.decode(data) as? [String: Any]
-      XCTAssertEqual(outputMap?["result"] as! NSNull, NSNull())
-      XCTAssertNil(outputMap?["error"])
+      let outputList = binaryMessenger.codec.decode(data) as? [Any]
+        XCTAssertEqual(outputList?.first as! NSNull, NSNull())
```

Comment:
Should we have an empty element for the error value?

Reply:
The array length is 1 if there is no error, so it would crash if we tried to check anything else.